www/install/insertBaseConf.sql

```diff
@@ -235,8 +235,8 @@ INSERT INTO `giv_graphs_template` (`graph_id`, `name`, `vertical_label`, `width`
 -- Contenu de la table Connector
 --
 INSERT INTO `connector` (`id`, `name`, `description`, `command_line`, `enabled`, `created`, `modified`) VALUES
-(1, 'Perl Connector', '', 'centreon_connector_perl', 1, UNIX_TIMESTAMP(), UNIX_TIMESTAMP()),
-(2, 'SSH Connector', '', 'centreon_connector_ssh', 1, UNIX_TIMESTAMP(), UNIX_TIMESTAMP());
+(1, 'Perl Connector', '', 'centreon_connector_perl --log-file=/var/log/centreon-engine/connector-perl.log', 1, UNIX_TIMESTAMP(), UNIX_TIMESTAMP()),
+(2, 'SSH Connector', '', 'centreon_connector_ssh --log-file=/var/log/centreon-engine/connector-ssh.log', 1, UNIX_TIMESTAMP(), UNIX_TIMESTAMP());
```

Comment:
a macro should be used here
cause `/var/log/centreon-engine` do not always exist (source install)

Reply:
I have searched for such macro. Actually it does only exist for centengine custom compilation, but when you install Centreon, the wizard does not offer to customize this path, unlike the broker log path. It seems to be hard-coded for the moment.

Reply:
during source install, this path is asked and set to a configuration file
when using RPM, this path is already set in the configuration file

Reply:
https://github.com/centreon/centreon/blob/develop/libinstall/functions#L712-L719

Reply:
to be tested, but it should work with : 
```
(1, 'Perl Connector', '', 'centreon_connector_perl --log-file=@monitoring_varlog@/connector-perl.log', 1, UNIX_TIMESTAMP(), UNIX_TIMESTAMP()),
(2, 'SSH Connector', '', 'centreon_connector_ssh --log-file=@monitoring_varlog@/connector-ssh.log', 1, UNIX_TIMESTAMP(), UNIX_TIMESTAMP());
```

Reply:
Ok thanks! I did not find it!